.github/workflows/publish.yaml

```
@@ -37,19 +35,36 @@ jobs:
     steps:
       - uses: actions/setup-node@v1
         with:
-          node-version: 15
+          node-version: 16
           registry-url: 'https://registry.npmjs.org'
+
       - name: Checkout
         uses: actions/checkout@v2
-        with:
-          fetch-depth: 1
+
+      - name: Setup env. variables
+        run: |
+        if [[ "${{ github.event_name }}" == "repository_dispatch" && "${{ github.event.action }}"== "build-images" ]]; then
+          echo "BEE_VERSION=${{ github.event.client_payload.tag }}" >> $GITHUB_ENV
+          echo "BUILD_IMAGE=true" >> $GITHUB_ENV
+          echo "COMMIT_VERSION_TAG=true" >> $GITHUB_ENV
+          echo "BUILD_IMAGE=false" >> $GITHUB_ENV
+          echo "STATE_COMMIT=true" >> $GITHUB_ENV
+        else
+          echo "BEE_VERSION=${{ github.event.inputs.beeVersion }}" >> $GITHUB_ENV
+          echo "BUILD_IMAGE=${{ github.event.inputs.beeVersionAsCommitHash }}" >> $GITHUB_ENV
+          echo "COMMIT_VERSION_TAG=${{ github.event.inputs.commitVersionTag }}" >> $GITHUB_ENV
+          echo "BUILD_IMAGE=${{ github.event.inputs.beeVersionAsCommitHash }}" >> $GITHUB_ENV
+          echo "STATE_COMMIT=${{ github.event.inputs.stateCommit }}" >> $GITHUB_ENV
+        fi
+
       - name: Auth to Github Package Docker Registry
-        if: ${{ github.event.inputs.buildImage == 'true' && success() }}
+        if: ${{ ithub.event_name == 'repository_dispatch' || (github.event.inputs.buildImage == 'true' && success())  }}
```

I guess it is a typeo
```suggestion
        if: ${{ github.event_name == 'repository_dispatch' || (github.event.inputs.buildImage == 'true' && success())  }}
```

---

Thx!